js/_components/_hide.js

```diff
@@ -1,23 +1,7 @@
-var hideSettings
-chopstick.hide =
-{
-    settings:
+export default function hide(hide = $('.js-hide')) {
```

Comment:
👍  I like the modularity with the parameters, so you can apply the hide function on other things than the default js-hide selector.